scripts/generate-manifest.js

```diff
@@ -53,38 +53,49 @@ function generateCatalogueManifest(opts) {
  * @param {string} [opts.version='0'] - Only include layers that satisfy this semver version
  * @param {boolean} [opts.production=false] - If true, include only production layers
  * @param {string} [opts.hostname=`${constants.VECTOR_STAGING_HOST}`] - Hostname for files in manifest
+ * @param {Object} [opts.fieldInfo=null] - Field metadata
  */
 function generateVectorManifest(sources, opts) {
   opts = {
     version: 'v0',
     production: false,
-    hostname: constants.VECTOR_STAGING_HOST, ...opts,
+    hostname: constants.VECTOR_STAGING_HOST,
+    fieldInfo: null, ...opts,
```

Comment:
opts shouldn't be on the same line - confusing
```suggestion
    fieldInfo: null,
    ...opts,
```